pydis_site/apps/content/views/articles.py

```diff
@@ -0,0 +1,18 @@
+from django.core.handlers.wsgi import WSGIRequest
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.views import View
+
+from pydis_site.apps.content.utils import get_articles, get_categories
+
+
+class ArticlesView(View):
+    """Shows all content and categories."""
+
+    def get(self, request: WSGIRequest) -> HttpResponse:
+        """Shows all content and categories."""
+        return render(
+            request,
+            "content/articles.html",
+            {"content": get_articles(), "categories": get_categories()}
+        )
```

Comment:
I'd use a `TemplateView` here and define `get_context_data`:

https://docs.djangoproject.com/en/3.1/topics/class-based-views/generic-display/#adding-extra-context